less/components/sidebar.less

```diff
@@ -55,6 +56,19 @@ html.no-js .sidebar {
   a {
     color: inherit;
   }
+
+  .collapsible-content {
+    transition: all 0.5s ease;
+    opacity: 1;
+    max-height: 1400px;
```

Comment:
Actually, I am trying out the animation now and I think it's fine as-is?

Here it is online (for a very short time) if you or @iamjessklein or @vazquez want to try it out: http://mozteach.toolness.org/